aten/src/ATen/native/nested/cuda/NestedTensorTransformerFunctions.cpp

```diff
@@ -101,5 +102,102 @@ Tensor nested_from_padded_cuda(
     return at::native::nested_from_padded_generic(padded, sizes);
   }
 }
+
+Tensor _collapse_two_dims_3(const Tensor& input, int64_t dim1, int64_t dim2) {
+  TORCH_CHECK(dim1 > 0, "dim1: Cannot collapse dim 0.");
```

Comment:
do you expect to accept different args in future? Now it looks like the only valid values are `1` and `2` for dim1 and dim2, so doesn't make sense to pass them as args.

Reply:
Yes, I do. It's probably better to make these comments...